src/parser/jobs/mnk/modules/Brotherhood.tsx

```diff
@@ -0,0 +1,89 @@
+import {t, Trans} from '@lingui/macro'
+import {ActionLink} from 'components/ui/DbLink'
+import {StatusRoot} from 'data/STATUSES'
+import {Event, Events} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {filter, oneOf} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import Checklist, {Requirement, Rule} from 'parser/core/modules/Checklist'
+import {Data} from 'parser/core/modules/Data'
+import React from 'react'
+import {DISPLAY_ORDER} from './DISPLAY_ORDER'
+
+const FULL_PARTY_SIZE = 8
+
+export class Brotherhood extends Analyser {
+	static override debug = false
+	static override handle = 'brotherhood'
+	static override title = t('mnk.bh.title')`Brotherhood`
+	static override displayOrder = DISPLAY_ORDER.BROTHERHOOD
+
+	@dependency private checklist!: Checklist
+	@dependency private data!: Data
+
+	private brotherhoodBuffTimestamps: number[] = []
+	private brotherhoodCastTimestamps: number[] = []
```

Comment:
I think these could use some renaming or otherwise rethink how we store hits. Probably like `brotherhoodUsed` and `brotherhoodApplied` or something. An interface approach is likely cleaner tho, see my comment on the `affectedPlayers` method.